### PR TITLE
Document latest docker file usage (user friendly)

### PR DESCRIPTION
**Pull request by lukegorman**
Additionally:

update getting started docs to use the new environment.yml over .txt :)

### CODE REVIEW

File: content/en/docs/Getting started/_index.md [hunk 1]
```diff
@@ -9,10 +9,9 @@ description: >
 ## Prerequisites
 
 This package requires;
-- python version 3.8+
-- Anaconda version 4.10+
-
-Are there any system requirements for using this project? What languages are supported (if any)? Do users need to already have any software or tools installed?
+- a linux environment or [docker](https://docs.docker.com/desktop/install/windows-install/) for Windows
+- python version 3.8.10
```
Comment (jpolton) [inline content/en/docs/Getting started/_index.md:13]: Is this a 3.8+ ?
We can't do 3.10 but we can do 3.9

File: content/en/docs/Getting started/_index.md [hunk 2]
```diff
@@ -9,10 +9,9 @@ description: >
...
+- [Miniconda](https://docs.conda.io/en/latest/miniconda.html#linux-installers)
```
Comment (jpolton) [inline content/en/docs/Getting started/_index.md:14]: `or Anaconda version 4.10+`
miniconda is a recommendation, not a requirement

File: content/en/docs/Getting started/_index.md (hunk 1 above)
Comment (jpolton) [reply content/en/docs/Getting started/_index.md:13]: ignore

File: content/en/docs/Getting started/_index.md (hunk 2 above)
Comment (jpolton) [reply content/en/docs/Getting started/_index.md:14]: ignore - given the Liverpool stuff is going elsewhere

**Review by jpolton — APPROVED** [review]

File: content/en/docs/Getting started/_index.md (hunk 1 above)
Comment (lukegorman) [reply content/en/docs/Getting started/_index.md:13]: lol

Comment (lukegorman) [reply content/en/docs/Getting started/_index.md:13]: COAsT really does need 3.8.................. point 10 not just any 3.8

**Review by jpolton — APPROVED** [review]

**Pull request merged**